data/tlg0591/1st1K002/tlg0591.1st1K002.1st1K-grc1.xml

```diff
@@ -4,10 +4,9 @@
   <teiHeader xml:lang="eng">
     <fileDesc>
   <titleStmt>
-    <title>Odysseus</title>
     <title xml:lang="grc">ΟΔΥΣΣΕΥΣ</title>
```

Comment:
Ὀδυσσεύς

Reply:
You know I think the Greek title was already in the file like that and I just deleted the English, but I will fix that! In general I'm trying not to add too many titles in any language that don't show up in the source itself!